climpred/utils.py

```diff
@@ -301,6 +340,54 @@ def _load_into_memory(res):
     return res
 
 
+def reduce_forecast_to_same_inits(forecast, verif):
```

Comment:
I'm not sure how this one popped back up, but this whole function should be deleted since it's handled in `alignment.py`.

Reply:
rebasing